lib/devtools.rb

```diff
@@ -15,6 +20,10 @@
 require 'devtools/project/initializer/rake'
 require 'devtools/project/initializer/rspec'
 
```

Comment:
Reasoning for these two was to separate 1) tool wrappers and 2) rake handlers.  Happy to remove though